evm-contracts/src/v0.7/dev/Operator.sol

```diff
@@ -339,19 +344,21 @@ contract Operator is
     uint256 nonce,
     uint256 dataVersion
   ) internal returns (bytes32 requestId, uint256 expiration) {
+    require(dataVersion < 2**8, "dataVersion too big");
```

Comment:
@se3000 This is casting to `uint8` from a `uint256` to make sure the Operator is backwards compatible with older versions of `ChainlinkClient` and the `ChainlinkRequestInterface`.

We could:

1. Keep like this.
2. Alter `ChainlinkRequestInterface` to define `uint8 dataVersion` instead of `uint256 dataVersion` in functions that require it. This would need to be done across the board (0.4, 0.5 & 0.6) and alter older versions of `ChainlinkClient` to be compatible with `Operator`.
3. Create a new `ChainlinkRequestInterface2` which does what number 2 does, requiring the same changes to older versions of `ChainlinkClient`.

Reply:
I like option 1. We have plenty of room to grow before it becomes an issue.